packages/ListBoxWithTags/test/ListBoxWithTags.spec.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { render, fireEvent, screen, waitFor } from "@testing-library/react"; // fireEvent, waitFor, screen
+import ListBoxTags from "../src";
+import Uncontrolled from "../stories/examples/Uncontrolled";
+
+function getTrigger(container) {
+  return container.querySelector("[data-pka-anchor='popover'] [role='button']");
+}
+
+function getPopover() {
+  return document.body.querySelector("[data-pka-anchor='popover.content']");
+}
+
+function clickListBoxWithTagOption(text) {
+  fireEvent.click(getPopover().querySelector(`li[value='${text}']`));
+}
+
+function getOptionInTrigger($trigger) {
+  return [...$trigger.querySelectorAll(`li`)].map(e => e.textContent);
+}
+
+function getOptionInPopover() {
+  return [...getPopover().querySelectorAll(`li`)].map(e => e.textContent);
+}
+
+function getDeleteButtonByOptionInTrigger(text, $trigger) {
+  return [...$trigger.querySelectorAll(`li`)]
+    .filter(li => li.textContent === text)[0]
+    .querySelector("[data-pka-anchor='list-box-tags.pill.delete']");
+}
+
+describe("ListBoxTags", () => {
+  it("should filter correctly the option", () => {
+    const data = [
+      { id: 1, label: "The Dog" },
+      { id: 2, label: "The Fox" },
+      { id: 3, label: "The Raccoon" },
+      { id: 4, label: "Dinosaur" },
+    ];
+
+    expect(ListBoxTags.filter("dog", data)).toMatchSnapshot();
+    expect(ListBoxTags.filter("fox", data)).toMatchSnapshot();
+    expect(ListBoxTags.filter("Raccoon", data)).toMatchSnapshot();
+    expect(ListBoxTags.filter("the", data)).toMatchSnapshot();
+    expect(ListBoxTags.filter("saur", data)).toMatchSnapshot();
```

Comment:
I thought we weren't using snapshots anymore

Reply:
these are not html snapshot these are javascript objects snapshots and are really handy when you want to assert that a specific object is returned whenever running a function. 

Instead of manually written the expected return object just assert that match the snapshot :)